job/client/src/main/java/alluxio/worker/job/RetryHandlingJobMasterClient.java

```diff
@@ -60,12 +60,6 @@ protected long getServiceVersion() {
     return Constants.JOB_MASTER_WORKER_SERVICE_VERSION;
   }
 
-  @Override
```

Comment:
These lines were added to get around not being able to grab configuration in beforeConnect() in the AbstractClient.java